platform/platform-scheduler/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -20,10 +20,17 @@
         description="Shell command to be used within the container. For example, log:set DEBUG">
     </AD>
     
-    <AD name="Interval in seconds" id="intervalInSeconds"
-        description="The interval of time in seconds between each execution. Must be a positive integer. For example, 3600 is 1 hour."
-        required="true" type="Integer"/>
-    
+    <AD name="Interval" id="intervalString"
+        description="The Interval String for each execution.  Based on the Interval Type, this will either be a Cron String, or a Second Interval.  (e.x. '0 0 0 1/1 * ? *' or '12')"
```

Comment:
1. You have two spaces after "...execution."
2. Remove comma after "...Cron String"